Source/WebCore/platform/graphics/SourceBufferPrivate.h

```diff
@@ -129,6 +130,8 @@ class SourceBufferPrivate
     WEBCORE_EXPORT void setClient(SourceBufferPrivateClient&);
     WEBCORE_EXPORT void detach();
 
+    void setMediaSourceDuration(const MediaTime& duration) { m_mediaSourceDuration = duration; }
```

Comment:
Could be MediaTime, since it is basically smaller than a uint64_t

Reply:
not really.
a MediaTime is :
```
    union {
        int64_t m_timeValue;
        double m_timeValueAsDouble;
    };
    uint32_t m_timeScale;
    uint8_t m_timeFlags;
```
so it's 16 bytes as uint8_t would take a word.